pandas/tests/reshape/test_pivot.py

```diff
@@ -1705,9 +1705,21 @@ def test_crosstab_with_numpy_size(self):
         tm.assert_frame_equal(result, expected)
 
     def test_crosstab_dup_index_names(self):
-        # GH 13279, GH 18872
+        # duplicated index name should get renamed (GH 19029)
         s = pd.Series(range(3), name='foo')
-        pytest.raises(ValueError, pd.crosstab, s, s)
+        failed = False
+        try:
```

Comment:
if you are asserting that this works, simply collect the result and compare vs expected